app/services/participant_declarations/mark_as_paid.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ParticipantDeclarations
+  class MarkAsPaid
+    def initialize(statement)
+      self.statement = statement
+    end
+
+    def call(participant_declaration)
+      ParticipantDeclaration.transaction do
+        participant_declaration.make_paid!
```

Comment:
as this has been extracted out and can therefore be called by anyone i personally would opt to add some guards here against someone who accidentally calls this class with a declaration which may not be in the right state to use this class. or maybe it's not a problem and has already been guarded elsewhere if let's say someone passed in a `submitted` or `awaiting_clawback` declaration?

Reply:
We don't really have proper statemachine here, normally this would raise an error if you aren't allowed this state transition, but unfortunately, these fail silently (`make_paid!` only does it when `payable?`)

Reply:
@asmega I've added a `line_item.payable?` guard close and some specs  around that